EarlGrey/Exception/GREYDefaultFailureHandler.m

```diff
@@ -52,11 +52,12 @@ - (void)handleException:(GREYFrameworkException *)exception details:(NSString *)
   }
 
   NSMutableString *log = [[NSMutableString alloc] init];
-  // Start on fresh new line.
-  [log appendString:@"\n"];
-  [log appendFormat:@"Exception: %@\n", exception.name];
   NSString *reason =
       exception.reason.length > 0 ? exception.reason : @"exception.reason was not provided";
+  [log appendFormat:@"%@\n\n", reason];
+  [log appendFormat:@"Bundle ID: %@\n\n", [[NSBundle mainBundle] bundleIdentifier]];
```

Comment:
It would help in case of looking at the app / test in case of multiprocess if I'm right.